app/verification.py

```diff
@@ -234,3 +282,84 @@ def get_website_verification(appid: str):
         }
 
     return _check_website_verification(appid)
+
+
+def _verify_app(appid: str, login, verified: bool):
+    """Marks a flatpak as verified. This requires the logged in user to have an account with the correct provider and
+    with the correct username, as specified by the app ID.
+
+    Returns: Same as get_verification_status()."""
+
+    if not login["state"].logged_in():
+        raise HTTPException(status_code=403, detail="not_logged_in")
+
+    if detail := _check_app_id_error(appid):
+        return {
+            "verified": False,
+            "method": "none",
+            "detail": detail,
+        }
+
+    username = _get_github_username(appid)
+    if username is None:
+        raise HTTPException(status_code=400, detail=ErrorDetail.INVALID_USERNAME)
+
+    accounts = sqldb.session.query(models.GithubAccount).filter_by(
+        user=login["user"].id
+    )
```

Comment:
The `GithubAccount` model has a `by_user()` method which does this.  It may be sensible to use that method.

It's intended to only ever have a single github account for a given flathub user, unless we want to change that.